src/components/Footer.tsx

```diff
@@ -10,7 +10,7 @@ const Copyright = () => {
 
 const Footer: React.FC = () => {
   return (
-    <footer className="border-color container mt-auto border-t py-3 sm:py-5">
```

Comment:
could you revert your changes here as well there was nothing changed except the ordering of the classes

Reply:
Sorry, the extension I used changed the ordering automatically